resources/js/i18n/en.json

```diff
@@ -761,16 +769,40 @@
         },
         "importer": {
             "title": "Data-Importer",
+            "entity_settings": "Entity Settings",
+            "attribute_mapping": "Attribute Mapping",
             "upload_csv_info": "Select a CSV file to import data.",
             "drop_csv_file": "Drop CSV file here",
+            "csv_file_error": "CSV-Datei konnte nicht gelesen werden.",
```

Comment:
english

Reply:
Fixed in e1f017ea68742e1c0d73bf36848c7e753edfb3c9